### ISSUE TITLE
A credential file (a.k.a .clasprc.json) permission is unsafe. (-rw-r--r--)

### ISSUE THREAD

**Issue by prprhyt** [google/clasp#387]
.clasprc.json has an access token.  
So, I expect to limit .clasprc.json permissions to only the owner after this file was created.

## Expected Behavior
```
$ ls -g .clasprc.json 
-rw-------  1 staff  583 11 11 23:30 .clasprc.json
```

## Actual Behavior
```
$ ls -g .clasprc.json 
-rw-r--r--  1 staff  583 11 11 23:30 .clasprc.json
```

## Specifications

  - Node version (`node -v`): v10.8.0
  - Version (`clasp -v`): 1.6.3
  - OS (Mac/Linux/Windows): macOS High Sierra 10.13.6

I checked [src/auth.ts](https://github.com/google/clasp/blob/8d4eceab9ebf64d38b9581a6d896d317f35c93bf/src/auth.ts) and [src/dotfile.ts](https://github.com/google/clasp/blob/8d4eceab9ebf64d38b9581a6d896d317f35c93bf/src/dotfile.ts).  
It looks like to write/read credentials using [dotf](https://github.com/grant/dotf/).   
In this case, which should I send Pull Requests, clasp or dotf?

**Comment by grant** [comment]
Create an issue and test case in dotf. Please send a PR there.

**Comment by prprhyt** [comment]
OK.  
I create an issue https://github.com/grant/dotf/issues/1 .

**Comment by grant** [comment]
Thanks for the PR. I've published the new version of dotf: `dotf@1.1`
Please update clasp's dependency to dotf and verify that it works in a PR.

**Comment by grant** [comment]
Will be on npm when I publish 2.0 next month.
